pkg/scheduler/framework/plugins/podtopologyspread/filtering_test.go

```diff
@@ -1651,7 +1652,7 @@ func TestPreFilterDisabled(t *testing.T) {
 	p := &PodTopologySpread{}
 	cycleState := framework.NewCycleState()
 	gotStatus := p.Filter(context.Background(), cycleState, pod, nodeInfo)
-	wantStatus := framework.NewStatus(framework.Error, `error reading "PreFilterPodTopologySpread" from cycleState: not found`)
+	wantStatus := framework.AsStatus(fmt.Errorf(`error reading "PreFilterPodTopologySpread" from cycleState: %w`, fmt.Errorf("not found")))
```

Comment:
what's the difference with this? anything more bring us?
```golang
framework.AsStatus(fmt.Errorf(`error reading "PreFilterPodTopologySpread" from cycleState: not found`))
```

Reply:
I was following it up from another PR..but I agree and will revert this back

Reply:
Done